unit-tests/src/test/java/org/eclipse/collections/impl/list/immutable/AbstractImmutableListTestCase.java

```diff
@@ -667,6 +668,58 @@ public void collectWithIndexWithTarget()
                 setOfPairs.collect(ObjectIntPair::getOne, Sets.mutable.empty()));
     }
 
+    /**
+     * @since 11.0.
```

Comment:
Do we need @ since tag in test methods?

Reply:
They're not strictly necessary but also don't hurt to have IMO.